src/node.cc

```diff
@@ -2843,6 +2844,11 @@ void SetupProcessObject(Environment* env,
     READONLY_PROPERTY(process, "_print_eval", True(env->isolate()));
   }
 
+  // -c, --check
+  if (syntax_check_only) {
+    READONLY_PROPERTY(process, "_syntax_check_only", True(env->isolate()));
```

Comment:
I would prefer using consistent camel casing. I am not sure if we can fix the `_print_eval` above. cc @bnoordhuis